### ISSUE TITLE
Text font adversely affects performance

### ISSUE THREAD

**Issue by ccbtag** [ChordPro/chordpro#141]
**Describe the bug**
Text font adversely affects the time it takes to generate a PDF.

- Courier takes ~1.4 seconds
- Georgia takes ~3.5 seconds
- Helvetica takes ~1.5 seconds
- Times-Roman takes ~1.5 seconds
- Verdana takes ~14 seconds 

**To Reproduce**

Create a text file `test.chordpro` with the following text copy and pasted 20+ times. Please note that this isn't reproducible without the embedded html.
```
<b>Chorus 1</b>
As the deer pants for the water
```

Generate a PDF:
```
# time chordpro --output=test.pdf --text-font=Verdana test.chordpro

real    0m7.801s
user    0m7.670s
sys     0m0.115s

# time chordpro --output=test.pdf --text-font=Helvetica test.chordpro

real    0m0.548s
user    0m0.524s
sys     0m0.023s
```

**Expected behavior**
I expect the performance to be consistent regardless of selected font.

**Please complete the following information:**
 - OS? CentOS Linux release 7.8.2003
 - Perl version? v5.16.3
 - ChordPro version? 0.977
 - Where did you get ChordPro from? CPAN

**Comment by sciurius** [comment]
Actually there is no reason to expect the performance to be consistent regardless of selected font. Some fonts are small, like the builtin fonts that have approx 100 glyphs. Some fonts, as --I assume-- Verdana, have thousands even tenthousands of glyphs so they require more time to process.

Having said that, there is something suboptimal going on as you can see from the file size. With one line of text, the file size becomes about 50k, with 20 repetitions of the same text it becomes 270k. This is due to using `<b></b>` which seems to bypass the font caching so there is a new Verdana-Bold font created for every occurrence of `<b></b>`.

Thanks for finding this, I'll investigate.

**Comment by sciurius** [comment]
This should be solved with ChordPro 0.979 and Text::Layout 0.021.
